docs/grammar.md

```diff
@@ -289,3 +289,16 @@ Note that `%ignore` directives cannot be imported. Imported rules will abide by
 
 Declare a terminal without defining it. Useful for plugins.
 
+### %override
+
+Override a rule, affecting all the rules that refer to it.
+
+Useful for implementing an inheritance pattern when importing grammars.
+
+**Example:**
+```perl
```

Comment:
Why `perl`? Shouldn't `lark` work?

Reply:
Good point! But vscode (or other editors) still can't highlight lark inside markdown

Reply:
But it's worth considering. In that case we'll have to change it all over the documentation.